connectors/cli/connector.py

```diff
@@ -30,7 +30,8 @@ async def list_connectors(self):
         # TODO move this on top
         try:
             await self.es_management_client.ensure_exists(
-                indices=[CONCRETE_CONNECTORS_INDEX, CONCRETE_JOBS_INDEX]
+                indices=[CONCRETE_CONNECTORS_INDEX, CONCRETE_JOBS_INDEX],
+                expand_wildcards="all",
```

Comment:
I think don't need `expand_wildcards` anymore because connectors work only with one index. 

I'm getting rid of the wildcards in this [PR](https://github.com/elastic/connectors/pull/1959/files)

Reply:
But this one is creating jobs and connectors indices that are hidden, I believe?

Reply:
I think `expand_wildcards` is needed only when you want to update multiple indices that match the pattern.  

Here, we operate only with concrete indices.